core/store/lib/search/adapter/graphql/queries/cmsBlock.gql

```diff
@@ -0,0 +1,13 @@
+query cmsBlocks ($filter: CmsInput) {
```

Comment:
should be in module/queries

Reply:
This gql is related to the ceratin searchAdapter. Not sure if it should be in the module/queries

Reply:
I agree with @filrak; if there is currently no way to fully add query/resultAdapter from the module level - then we need to extend the graphQL API to have this capability. Anyway - everything should be registered from the module level not modifying the core; CMS module should be a example how decoupled the app is with new module system

Reply:
ok i see the idea. then probably it makes sense 
i will try move register entity Type to the module level